data/jobs.txt

```diff
@@ -1379,6 +1379,80 @@ mission "Bounty Hunting (Small)"
 		payment 150000
 		dialog "The government of <planet> gratefully pays you <payment> for eliminating the <npc>."
 
+mission "Bounty Hunting (Small, boarding, entering)"
+	name "Board <npc>"
+	description "A wanted criminal is fleeing in a stolen hot rod ship, the <npc>, which is about to enter this system. Disable and board the vessel and bring the criminal to <destination>. Reward for capture: <payment>."
+	passengers 1
+	repeat
+	job
+	to offer
+		"combat rating" > 80
+		random < 15
+	on offer
+		require "Brig"
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral"
```

Comment:
Should we start making jobs that can serve or be served from "Independent" planets?

Reply:
That sounds like a good idea, but maybe do that in a separate PR?

Reply:
Your new jobs can start the trend :+1:

Reply:
Sounds good!